exercism/bracket-push/bracket-push.js

```diff
@@ -0,0 +1,6 @@
+var bracket = function(input) {
+  //define what ending bracket should be if the beginning is found
+};
+
```

Comment:
You should try to keep incomplete placeholders like this out of commits for complete tests, that way you have more freedom to cherry pick commits later or reorder them during a rebase without ending up with files getting reverted to a broken state.

Personally I try to make a separate commit for each change (ex: secret handshake would be a different commit from bob, and bob's commit would never have changes to secret handshake's files) but that's just one way of keeping git stuff organized.